### PR TITLE
Enable utility features

### PR DESCRIPTION
**Pull request by seethatgo**
This is the initial impelmentation of the utility feature based on the requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (ademotest) [inline util/config.txt:3]: This config value should be 567
123 was the old value

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
```
Comment (ademotest) [inline util/feature.txt:3]: Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgits.

**Review by ademotest — CHANGES_REQUESTED** [review]
Please fix the config values and add the implementation to feature2

File: util/config.txt (hunk 1 above)
Comment (seethatgo) [reply util/config.txt]: added correct value

File: util/feature.txt (hunk 2 above)
Comment (seethatgo) [reply util/feature.txt:3]: added additional functionality

File: util/config.txt (hunk 1 above)
Comment (seethatgo) [reply util/config.txt]: @ademotest see the latest commit for the change

**Review by ademotest — APPROVED** [review]

**Pull request merged**